GCMtools/utils/interface.py

```diff
@@ -10,27 +10,344 @@
 #        - ...
 # ==============================================================
 
+import numpy as np
+import xarray as xr
+
+from ..core.const import VARNAMES as c
+
+
+class Chemistry:
+    """
+    Chemistry class used to deal with different kinds of chemical models.
+    """
+    abunds = xr.Dataset()
```

Comment:
This needs to go into the constructor, otherwise it is a 'class variable' instead of an 'instance variable'.